autoware_adapi_v1_msgs/vehicle/msg/VehicleDoor.msg

```diff
@@ -0,0 +1,18 @@
+# constants for door status
+uint8 UNKNOWN = 0
+uint8 DOOR_OPENED = 1
+uint8 DOOR_CLOSED = 2
+uint8 DOOR_OPENING = 3
+uint8 DOOR_CLOSING = 4
+uint8 NOT_APPLICABLE = 5
+
+# constants for door locations
+uint8 FRONT = 1
+uint8 BACK = 2
+uint8 MIDDLE = 3
+uint8 LEFT = 4
+uint8 RIGHT = 5
+
+uint8 door_id
+uint8 status
+uint8[] door_location
```

Comment:
Door locations will be provided in [the vehicle info API](https://github.com/autowarefoundation/autoware/issues/3226). So `door_id` and `door_location` can be removed by using the array index.

Reply:
I see, so the array index will define the door status.
Okay I will remove it

Reply:
changed at [4b18f1e](https://github.com/autowarefoundation/autoware_adapi_msgs/pull/24/commits/4b18f1ebcfeb7cbf6a14f44d943abd9e80617538)